fireant/tests/widgets/test_pandas.py

```diff
@@ -30,6 +30,9 @@ def _format_float(x):
     return '{:,.0f}'.format(x)
 
 
+pd.set_option('display.max_columns', 500)
+pd.set_option('display.width', 1000)
```

Comment:
this is inconsequential. Makes it a bit easier to debug these tests, so can just leave it in the module.